packages/dd-trace/src/plugins/util/web.js

```diff
@@ -270,15 +270,26 @@ function wrapEnd (req) {
     return returnValue
   }
 
-  Object.defineProperty(res, 'end', {
-    configurable: true,
-    get () {
-      return this._datadog_end
-    },
-    set (value) {
-      this._datadog_end = scope.bind(value, req._datadog.span)
-    }
-  })
+  res._datadog_req = req
+  res._datadog_scope = scope
```

Comment:
Why is this needed since `scope` is basically global?

Reply:
I was replicating the previous code which grabbed it from the local scope.

That being said, you're right. No need to stash this.